.github/workflows/test-links.yml

```diff
@@ -0,0 +1,19 @@
+# used for running tests on links
```

Comment:
can you be a bit more verbose on what you mean by "links"? Also, it would be helpful to add justification

Reply:
Thanks! Yes - I'll add in further notes here.